### ISSUE TITLE
Remove legacy python2 dependencies

### ISSUE THREAD

**Issue by egede** [ganga-devs/ganga#2222]
In a few helper and template scripts we in fact still use `python2`. This should be updated
```
GangaCore/Lib/Batch/BatchScriptTemplate.py.template:#!/usr/bin/env python2
GangaCore/Lib/Notebook/Notebook.py:                defvalue='python2',
GangaCore/test/GPI/Notebook/Test.ipynb:   "name": "python2"
GangaCore/test/GPI/Notebook/Test.ipynb:   "pygments_lexer": "ipython2",
GangaDirac/Lib/RTHandlers/ExeDiracRTHandler.py:    script_template = """#!/usr/bin/env python2
GangaLHCb/Lib/RTHandlers/ExeDiracRTHandler.py:    script_template = """#!/usr/bin/env python2
```

**Comment by souravpy** [comment]
Should this be updated to latest python version 3.14?

**Comment by egede** [comment]
> Should this be updated to latest python version 3.14?

No, I think it should just be updated to say `python3` instead of `python2`. And the scripts should be updated if they have something in them that is not python3 valid syntax. These are simple wrapper scripts, so they should be kept as "version agnostic" as possible as we do not have control of the exact version used (could be anything at the moment between 3.6 and 3.14).

**Comment by mesmith75** [comment]
Fixed by #2269
